code/projects/steg/src/lib/src/image_data.rs

```diff
@@ -1,7 +1,6 @@
-use wasm_bindgen::Clamped;
-use web_sys::ImageData;
+use web_sys::{js_sys::Uint8ClampedArray, ImageData};
 
 pub fn create_image_data(data: &[u8], width: u32) -> ImageData {
-    ImageData::new_with_u8_clamped_array(Clamped(data), width)
+    ImageData::new_with_js_u8_clamped_array(&Uint8ClampedArray::from(data), width)
```

Comment:
**note:** Fixes the "detached buffer" bug (or rather avoids it, since it's a bug in `web_sys` and not our code, see https://github.com/rustwasm/wasm-bindgen/issues/2445 and https://github.com/rustwasm/wasm-bindgen/issues/423).